test/cpp/jit/tests.h

```diff
@@ -867,11 +866,11 @@ void testDifferentiateWithRequiresGrad(std::ostream& out = std::cout) {
   PropagateRequiresGrad(graph);
 
   auto grad_spec = differentiate(graph);
-  std::vector<size_t> expected_input_vjps = {1, 2}; // for e and %4 = (d + a)
+  std::vector<size_t> expected_input_vjps = {1, 4}; // for e and %4 = (d + a)
```

Comment:
The extra 2 are from saving the sizes, right? It might be good to write that in a comment here

Reply:
It's only an increased offset, the comment in here is up to date. I'm not sure if it's worth adding a comment there, since it's going to be the standard practice to carry over a ton of sizes to backward from now on.